sdk/identity/identity/review/identity.api.md

```diff
@@ -42,14 +43,15 @@ export class AzureCliCredential implements TokenCredential {
     constructor();
     protected getAzureCliAccessToken(resource: string): Promise<unknown>;
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
-}
+    }
```

Comment:
I'm not sure what's up with these lines. I'm assuming this is generated by something outside of my changes.

Reply:
api-extractor has been pinned to `7.7.11` for quite a while now:

https://github.com/Azure/azure-sdk-for-js/blob/7dbbf0bcce29b0a5cc0d94a0ccb89dfdcca5b3ae/sdk/core/core-http/package.json#L149

Reply:
This file is only changed by api-extractor. Weirdly enough, if I rebuild things on master, this doesn't change. Mmmm